docs/reference/query-dsl/geo-shape-query.asciidoc

```diff
@@ -56,24 +56,17 @@ The following query will find the point using the Elasticsearch's
 --------------------------------------------------
 GET /example/_search
 {
-    "query":{
-        "bool": {
-            "must": {
-                "match_all": {}
-            },
-            "filter": {
-                "geo_shape": {
-                    "location": {
-                        "shape": {
-                            "type": "envelope",
-                            "coordinates" : [[13.0, 53.0], [14.0, 52.0]]
-                        },
-                        "relation": "within"
-                    }
-                }
-            }
-        }
+  "query": {
+    "geo_shape": {
+      "geo": {
+        "shape": {
+          "type": "envelope",
```

Comment:
This looks wrong? why are we changing this?

Reply:
I tested & it works. I proposed this change because enclosing the query within:

```"bool": {
            "must": {
                "match_all": {}
            },
            "filter": {
...
}}}}```  

is not needed, and in the documentation this distracts from the DSL query syntax.

I can rollback &/or move to a new PR please confirm

Reply:
We can do that in a follow up PR, let's just add the new changes so it is easy to review.

Reply:
restored